app/mailers/devise_mailer.rb

```diff
@@ -5,5 +5,5 @@ class DeviseMailer < Devise::Mailer
   include Roadie::Rails::Automatic
   include Devise::Controllers::UrlHelpers
 
-  default from: 'Pupilfirst <noreply@pupilfirst.com>'
+  default from: "< #{ENV['DEFAULT_SENDER_EMAIL']}>"
```

Comment:
Read from `Rails.application.secrets.default_sender_address` instead. It's cleaner to read from environment in one place (`secrets.yml`), and to refer to that central location for config, when required.

Also, there is an extra space after the `<` character here.